addons/hr/views/hr_department_views.xml

```diff
@@ -75,77 +75,59 @@
-                                        <div class="text-muted">
-                                            <field name="manager_id" widget="many2one_avatar_user" options="{'display_avatar_name': True}" readonly="1"/>
-                                        </div>
-                                        <div class="o_secondary" groups="base.group_multi_company">
-                                            <small>
-                                                <i class="fa fa-building-o" role="img" aria-label="Company" title="Company"/> <field name="company_id"/>
-                                            </small>
-                                        </div>
-                                    </div>
+                        <t t-name="kanban-card">
+                            <a type="open"><field name="name" class="fw-bold fs-4 d-bolck ms-2"/></a>
+                            <field name="manager_id" widget="many2one_avatar_user" options="{'display_avatar_name': True}" readonly="1" class="text-muted ms-2"/>
+                            <div class="small mt-1 ms-2" groups="base.group_multi_company">
+                                <i class="fa fa-building-o" role="img" aria-label="Company" title="Company"/> <field name="company_id"/>
+                            </div>
+                            <div class="row g-0 mt-3 mb-2 ms-2" t-if="!selection_mode">
+                                <div class="col-6 o_kanban_primary_left">
+                                    <button class="btn btn-primary" name="%(act_employee_from_department)d" type="action">
+                                        <field name="total_employee"/> Employees
+                                    </button>
                                 </div>
-                                <div class="container o_kanban_card_content" t-if="!selection_mode">
-                                    <div class="row o_kanban_card_upper_content">
-                                        <div class="col-6 o_kanban_primary_left">
-                                            <button class="btn btn-primary" name="%(act_employee_from_department)d" type="action">
-                                                <t t-out="record.total_employee.raw_value"/> Employees
-                                            </button>
-                                        </div>
-                                        <div class="col-6 o_kanban_primary_right">
-                                        </div>
-                                    </div>
-                                    <div class="o_kanban_card_lower_content"
-                                         style="justify-content: end">
-                                        <!-- placeholder for bottom content -->
-                                    </div>
+                                <div class="col-6 o_kanban_primary_right">
```

Comment:
Actually, could you use a `name` instead of a `class` ? If it's only for xpaths... Same below for the lower_content stuff